.astyleignore

```diff
@@ -1,2 +1,3 @@
 libraries/
+libs/
```

Comment:
what is this directory is used for?

Reply:
When I built with `mbed` CLI it created git repositories under `libs/`, but I can remove it